openml-lightgbm-module/pom.xml

```diff
@@ -19,21 +19,15 @@
     </parent>
     <packaging>pom</packaging>
 
-    <groupId>com.feedzai.openml.lightgbm.meta</groupId>
-    <artifactId>openml-lightgbm-meta-module</artifactId>
+    <groupId>com.feedzai.openml.lightgbm</groupId>
+    <artifactId>openml-lightgbm-module</artifactId>
     <description>OpenML LightGBM Machine Learning Model and Classifier</description>
 
     <modules>
         <!-- Builds LightGBM + lightgbmlib.jar -->
-        <module>lightgbm-builder</module>
+        <module>lightgbm-lib</module>
+
         <!-- The provider itself, which requires such artifacts -->
         <module>openml-lightgbm</module>
     </modules>
-
-    <properties>
```

Comment:
I'd rather see this properties here at the module level, and then lightgbm-lib use those properties to build the appropriate version according to this properties, and lightgbm just read this same version as well, instead of re-declaring versions in 3 places: properties, and each of the 2 child poms.

This way we only declare lightgbm-lib version once.